src/main/java/Entities/RecipeBook.java

```diff
@@ -1,160 +1,204 @@
-package main.java.Entities;

```

Comment:
There is an updated file in Pull Request #81 containing the updated sorted methods and camel case fixes. The added codes in this file look good and can be added to the updated files that were merged into main.